src/lib/models/ConfigModel.ts

```diff
@@ -23,7 +23,6 @@ export interface ConfigModel {
 }
 
 export class AnnotatedConfigModel implements ConfigModel {
-	private readonly config: CodeAnalyzerConfig; // TODO: It would be nice if we updated the CodeAnalyzer (in our core module) to just return its CodeAnalyzerConfig with a getter so we didn't need to pass it around
```

Comment:
With the other enhancements that I just pulled in, I was able to accomplish this TODO item as well.